src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin square {
+  width: $ceil-width;
+  height: $ceil-height;
+}
```

Comment:
It's better to accept width as a parameter, so in this case, you can create different squares